app/assets/stylesheets/views/_specialist-document.scss

```diff
@@ -1,3 +1,33 @@
+$contents-link-background: #dee0e2;
+
 .specialist-document {
   @include sidebar-with-body;
 }
+
+.govuk-sticky-element {
+  width: 100%;
+  margin: 0 $gutter-half;
+}
+
+.contents-list-link-wrapper {
```

Comment:
I think we should be able to move this styling into the back-to-top component? Then it could live with the rest of the `app-c-back-to-top` styles.